proxies/ruby/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.1 as builder
+FROM ruby as builder
```

Comment:
does this also pick latest stable?

Reply:
Yep - same rule as any other docker image